pages/products/index.tsx

```diff
@@ -0,0 +1,187 @@
+import Layout from "../../components/Layout";
+import { Wrapper } from "@cabindao/topo";
+import { styled } from "../../stitches.config";
+import Link from "next/link";
+import { ArrowRightIcon } from "@radix-ui/react-icons";
+import Image from "next/image";
+import ButtonLink from "../../components/ButtonLink";
+
+const Container = styled("section", {
+  pt: "$12",
+});
+
+const HeroContainer = styled("div", {
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "space-between",
+  height: 200,
+  position: "relative",
+  h1: {
+    position: "absolute",
+    left: 4,
+    top: "calc(50% - 22px)",
+    fontSize: 32,
+    lineHeight: "44px",
+    color: "$forest",
+    mt: 0,
+  },
+});
+
+const Row = styled("div", {
+  display: "flex",
+  justifyContent: "space-between",
+});
+
+const HeroElement = styled("div", {
```

Comment:
this might be overkill, but this makes a nested hexagon to make the pattern found in the designs. figured that multiple divs would be more performant than an image

Reply:
🤘 I dig it.